doc/src/manual/mathematical-operations.md

```diff
@@ -361,7 +361,6 @@ Julia applies the following order and associativity of operations, from highest
 | Bitshifts      | `<< >> >>>`                                                                                       | Left                       |
 | Addition       | `+ - \| ⊻`                                                                                        | Left[^2]                    |
 | Syntax         | `: ..`                                                                                            | Left                       |
-| Syntax         | `\|>`                                                                                             | Left                       |
 | Syntax         | `<\|`                                                                                             | Right                      |
```

Comment:
If `|>` is being removed, shouldn't `<|` also be removed here?

Reply:
Hmm. Not sure. Why?

Reply:
Oh I see. Yeah dunno.